index.html

```diff
@@ -69,12 +72,235 @@ <h1 class="col col-11 col-lg-5 aligne-self-center"> glade to see you</h1>
         </div>
     </section>
         
-    
-   
+    <section class="grid-container">
```

Comment:
Dear @m05e5,

Nice job is done! 👏👏👏

You have spaces keep in code, it will better you will consider it for more professional code view. 

Also, I fix that you have created two pull requested. You do not need to create the new one, just fix the reviewer changes and then after approval merges it.

Happy codes!!! 🎉 🥂

Elmar @elmar8287